server/graphql/invoice/src/mutations/customer_return/insert.rs

```diff
@@ -20,6 +21,13 @@ pub struct InsertInput {
     pub customer_id: String,
     pub outbound_shipment_id: Option<String>,
     pub customer_return_lines: Vec<CustomerReturnLineInput>,
+    pub status: Option<InsertCustomerReturnStatusInput>,
+}
+
+#[derive(Enum, Copy, Clone, PartialEq, Eq, Debug)]
+pub enum InsertCustomerReturnStatusInput {
+    New,
+    Shipped,
```

Comment:
Have not used the 'picked' status option for input. Maybe there is a future use case for this? I figure it is easy to add in future if need be.